sympy/sets/fancysets.py

```diff
@@ -393,9 +400,8 @@ class Range(Set):
         Range(0, 12, 3)
 
     Infinite ranges are allowed. ``oo`` and ``-oo`` are never included in the
-    set (``Range`` is always a subset of ``Integers``). If the starting point
```

Comment:
let's check to see that Range reports itself correctly as being a subset of Integers when it has non-Integer indices.

Reply:
Did you check it? If not then I will try to do so.

Reply:
I tried and found that it holds true for symbolic Range
```
>>> from sympy import *
>>> var('n, m', positive=True)
(n, m)
>>> Range(n, m).intersection(Integers)
Range(n, m, 1)
>>> Range(n, 1).intersection(Integers)
Range(n, 1, 1)
```